lib/index.d.ts

```diff
@@ -525,6 +525,22 @@ export interface CssNodeCommon {
     loc?: CssLocationRange | null;
 }
 
+export interface CssParentNodeCommon<T = CssNode> extends CssNodeCommon {
+    children: List<T>;
+}
+
+export interface CssParentNodeCommonPlain<T = CssNodePlain> extends CssNodeCommon {
+    children: T[];
+}
+
+export interface CssOptionalParentNodeCommon<T = CssNode> extends CssNodeCommon {
+    children: List<T> | null;
+}
+
+export interface CssOptionalParentNodeCommonPlain<T = CssNodePlain> extends CssNodeCommon {
+    children: T[] | null;
+}
```

Comment:
Maybe we could restrict the generic type parameters to so that these interfaces aren't accidentally used with `children` of a wrong type. It looks like the default type is already the most general type that would make sense.

```suggestion
export interface CssParentNodeCommon<T extends CssNode = CssNode> extends CssNodeCommon {
    children: List<T>;
}

export interface CssParentNodeCommonPlain<T extends CssNodePlain = CssNodePlain> extends CssNodeCommon {
    children: T[];
}

export interface CssOptionalParentNodeCommon<T extends CssNode = CssNode> extends CssNodeCommon {
    children: List<T> | null;
}

export interface CssOptionalParentNodeCommonPlain<T extends CssNodePlain = CssNodePlain> extends CssNodeCommon {
    children: T[] | null;
}
```